include/uv-nuttx.h

```diff
@@ -72,7 +71,7 @@
 #define TCP_NODELAY   1
 
 //-----------------------------------------------------------------------------
-// @20171130-sanggyu:
+// TUV_CHANGES@20171130
```

Comment:
It was my intention. I added ":" for separator for one line comment.
Do you want to add ":" for all tags?